package/MDAnalysis/transformations/__init__.py

```diff
@@ -26,34 +26,72 @@
 Trajectory transformations --- :mod:`MDAnalysis.transformations`
 ================================================================
 
-The transformations submodule contains a collection of functions to modify the
-trajectory. Coordinate transformations, such as PBC corrections and molecule fitting
-are often required for some analyses and visualization, and the functions in this
-module allow transformations to be applied on-the-fly.
-These transformation functions can be called by the user for any given
-timestep of the trajectory, added as a workflow using :meth:`add_transformations`
-of the :mod:`~MDAnalysis.coordinates.base` module, or upon Universe creation using
+The transformations submodule contains a collection of function-like classes to
+modify the trajectory.
+Coordinate transformations, such as PBC corrections and molecule fitting
+are often required for some analyses and visualization, and the functions in
+this module allow transformations to be applied on-the-fly.
+
+A typical transformation class looks like this:
+
+.. code-blocks:: python
+
+    class transformation(object):
+        def __init__(self, *args, **kwargs):
+            #  do some things
+            #  save needed args as attributes.
+            self.needed_var = args[0]
+
+        def __call__(self, ts):
+            #  apply changes to the Timestep object
+            return ts
+
+See `MDAnalysis.transformations.translate` for a simple example.
```

Comment:
The problem is they can't see the code from the docs, so instead have the above example be translate

Reply:
I think this part (inside `__init__`) of the doc is not present anywhere in the doc webpages. A more explicit description of how transformation is done is written separately. By which I guess makes it fine to be a little more abstract?